--- python/src/config_utils.py	2026-01-23 05:45:21.261688739 +0000
+++ temp_tools/config_utils.py	2026-01-23 05:45:21.245688701 +0000
@@ -26,6 +26,8 @@
         return 'lfm2'
     elif 'qwen' in model_type_str:
         return 'qwen'
+    elif 'moonshine' in model_type_str:
+        return 'moonshine'
     elif 'llama' in model_type_str:
         if output_dir and 'smol' in str(output_dir):
             return 'smol'
